samples/zephyr-hello/prj.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_CPLUSPLUS=n
+CONFIG_PRINTK=y
+CONFIG_BOOT_BANNER=y
+CONFIG_BUILD_OUTPUT_BIN=y
+CONFIG_NEWLIB_LIBC=y
```

Comment:
Why do you need libc here?

Reply:
Kotlin runtime uses finctions like `abort`, `ceilf` and so on, not available in default minimal Zephyr libc.
And AFAIK Zephyr itself plans to make newlib default choice.

Reply:
Don't know, if remove newlib there a lot of errors:
```
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:main(kotlin.Array<kotlin.String>)' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:main(kotlin.Array<kotlin.String>)'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.Any.toString()kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.Any.toString()kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KClassImpl.<get-qualifiedName>()kotlin.String?' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KClassImpl.<get-qualifiedName>()kotlin.String?'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KClassImpl.<get-simpleName>()kotlin.String?' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KClassImpl.<get-simpleName>()kotlin.String?'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.checkRadix$stdlib(kotlin.Int)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.checkRadix$stdlib(kotlin.Int)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.append(kotlin.String)kotlin.text.StringBuilder' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.append(kotlin.String)kotlin.text.StringBuilder'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.toString()kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.toString()kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.ThrowInvalidReceiverTypeException(kotlin.reflect.KClass<#STAR>)kotlin.Nothing' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.ThrowInvalidReceiverTypeException(kotlin.reflect.KClass<#STAR>)kotlin.Nothing'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.RuntimeException.<init>(kotlin.String?)kotlin.RuntimeException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.RuntimeException.<init>(kotlin.String?)kotlin.RuntimeException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.Exception.<init>(kotlin.String?)kotlin.Exception' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.Exception.<init>(kotlin.String?)kotlin.Exception'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.Throwable.<init>(kotlin.String?;kotlin.Throwable?)kotlin.Throwable' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.Throwable.<init>(kotlin.String?;kotlin.Throwable?)kotlin.Throwable'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.concurrent.FreezeAwareLazyImpl.<init>(kotlin.Function0<#GENERIC>)kotlin.native.concurrent.FreezeAwareLazyImpl<#GENERIC>' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.concurrent.FreezeAwareLazyImpl.<init>(kotlin.Function0<#GENERIC>)kotlin.native.concurrent.FreezeAwareLazyImpl<#GENERIC>'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.concurrent.FreezeAwareLazyImpl.toString()kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.concurrent.FreezeAwareLazyImpl.toString()kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.concurrent.FreezeAwareLazyImpl.<get-value>()#GENERIC' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.concurrent.FreezeAwareLazyImpl.<get-value>()#GENERIC'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.toString@kotlin.Any?.()kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.toString@kotlin.Any?.()kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.concurrent.Lock.lock()' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.concurrent.Lock.lock()'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.ThrowNullPointerException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.ThrowNullPointerException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.concurrent.Lock.unlock()' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.concurrent.Lock.unlock()'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.concurrent.InvalidMutabilityException.<init>(kotlin.String)kotlin.native.concurrent.InvalidMutabilityException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.concurrent.InvalidMutabilityException.<init>(kotlin.String)kotlin.native.concurrent.InvalidMutabilityException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.AssertionError.<init>(kotlin.Any?)kotlin.AssertionError' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.AssertionError.<init>(kotlin.Any?)kotlin.AssertionError'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.Error.<init>(kotlin.String?)kotlin.Error' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.Error.<init>(kotlin.String?)kotlin.Error'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.Throwable.toString()kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.Throwable.toString()kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.NullPointerException.<init>()kotlin.NullPointerException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.NullPointerException.<init>()kotlin.NullPointerException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.Exception.<init>()kotlin.Exception' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.Exception.<init>()kotlin.Exception'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.ensureCapacity(kotlin.Int)' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.ensureCapacity(kotlin.Int)'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.copyOfUninitializedElements$stdlib@kotlin.CharArray.(kotlin.Int;kotlin.Int)kotlin.CharArray' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.copyOfUninitializedElements$stdlib@kotlin.CharArray.(kotlin.Int;kotlin.Int)kotlin.CharArray'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.append(kotlin.Int)kotlin.text.StringBuilder' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.append(kotlin.Int)kotlin.text.StringBuilder'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.IllegalArgumentException.<init>(kotlin.String?)kotlin.IllegalArgumentException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.IllegalArgumentException.<init>(kotlin.String?)kotlin.IllegalArgumentException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.<init>(kotlin.Int)kotlin.text.StringBuilder' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.<init>(kotlin.Int)kotlin.text.StringBuilder'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.subSequence(kotlin.Int;kotlin.Int)kotlin.CharSequence' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.subSequence(kotlin.Int;kotlin.Int)kotlin.CharSequence'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.append(kotlin.Char)kotlin.text.StringBuilder' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.append(kotlin.Char)kotlin.text.StringBuilder'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.append(kotlin.CharSequence?;kotlin.Int;kotlin.Int)kotlin.text.StringBuilder' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.append(kotlin.CharSequence?;kotlin.Int;kotlin.Int)kotlin.text.StringBuilder'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.append(kotlin.CharSequence?)kotlin.text.StringBuilder' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.append(kotlin.CharSequence?)kotlin.text.StringBuilder'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.get(kotlin.Int)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.get(kotlin.Int)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.IndexOutOfBoundsException.<init>()kotlin.IndexOutOfBoundsException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.IndexOutOfBoundsException.<init>()kotlin.IndexOutOfBoundsException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.substring(kotlin.Int;kotlin.Int)kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.substring(kotlin.Int;kotlin.Int)kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.Int.rangeTo(kotlin.Int)kotlin.ranges.IntRange' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.Int.rangeTo(kotlin.Int)kotlin.ranges.IntRange'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.ranges.IntProgression.<init>$stdlib(kotlin.Int;kotlin.Int;kotlin.Int)kotlin.ranges.IntProgression' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.ranges.IntProgression.<init>$stdlib(kotlin.Int;kotlin.Int;kotlin.Int)kotlin.ranges.IntProgression'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.internal.getProgressionLastElement$stdlib(kotlin.Int;kotlin.Int;kotlin.Int)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.internal.getProgressionLastElement$stdlib(kotlin.Int;kotlin.Int;kotlin.Int)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.ranges.IntRange.toString()kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.ranges.IntRange.toString()kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.ranges.IntProgressionIterator.nextInt()ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.ranges.IntProgressionIterator.nextInt()ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.NoSuchElementException.<init>()kotlin.NoSuchElementException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.NoSuchElementException.<init>()kotlin.NoSuchElementException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.IntIterator.$<bridge-UNN>next()ValueType()kotlin.Any?' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.IntIterator.$<bridge-UNN>next()ValueType()kotlin.Any?'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.ranges.IntProgression.hashCode()ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.ranges.IntProgression.hashCode()ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.ranges.IntProgression.toString()kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.ranges.IntProgression.toString()kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.ranges.IntProgression.equals(kotlin.Any?)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.ranges.IntProgression.equals(kotlin.Any?)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.substringAfterLast$default@kotlin.String.(kotlin.String;kotlin.String;kotlin.Int)kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.substringAfterLast$default@kotlin.String.(kotlin.String;kotlin.String;kotlin.Int)kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.substringAfterLast@kotlin.String.(kotlin.String;kotlin.String)kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.substringAfterLast@kotlin.String.(kotlin.String;kotlin.String)kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.lastIndexOf$default@kotlin.CharSequence.(kotlin.String;kotlin.Int;kotlin.Boolean;kotlin.Int)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.lastIndexOf$default@kotlin.CharSequence.(kotlin.String;kotlin.Int;kotlin.Boolean;kotlin.Int)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.lastIndexOf@kotlin.CharSequence.(kotlin.String;kotlin.Int;kotlin.Boolean)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.lastIndexOf@kotlin.CharSequence.(kotlin.String;kotlin.Int;kotlin.Boolean)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.indexOf#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.indexOf#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.regionMatchesImpl$stdlib@kotlin.CharSequence.(kotlin.Int;kotlin.CharSequence;kotlin.Int;kotlin.Int;kotlin.Boolean)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.regionMatchesImpl$stdlib@kotlin.CharSequence.(kotlin.Int;kotlin.CharSequence;kotlin.Int;kotlin.Int;kotlin.Boolean)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KClassImpl.toString()kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KClassImpl.toString()kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.add(#GENERIC)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.add(#GENERIC)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.addAtInternal#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.addAtInternal#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.insertAtInternal#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.insertAtInternal#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.ensureCapacity(kotlin.Int)' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.ensureCapacity(kotlin.Int)'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.IllegalStateException.<init>()kotlin.IllegalStateException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.IllegalStateException.<init>()kotlin.IllegalStateException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.copyOfUninitializedElements$stdlib@kotlin.Array<#GENERIC>.(kotlin.Int;kotlin.Int)Generic' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.copyOfUninitializedElements$stdlib@kotlin.Array<#GENERIC>.(kotlin.Int;kotlin.Int)Generic'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.hashCode()ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.hashCode()ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.toString()kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.toString()kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.equals(kotlin.Any?)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.equals(kotlin.Any?)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.containsAll(kotlin.collections.Collection<#GENERIC>)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.containsAll(kotlin.collections.Collection<#GENERIC>)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.contains(#GENERIC)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.contains(#GENERIC)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.AbstractCollection.toArray(kotlin.Array<#GENERIC>)Generic' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.AbstractCollection.toArray(kotlin.Array<#GENERIC>)Generic'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.AbstractCollection.toArray()kotlin.Array<kotlin.Any?>' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.AbstractCollection.toArray()kotlin.Array<kotlin.Any?>'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.retainAll(kotlin.collections.Collection<#GENERIC>)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.retainAll(kotlin.collections.Collection<#GENERIC>)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.addAll(kotlin.collections.Collection<#GENERIC>)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.addAll(kotlin.collections.Collection<#GENERIC>)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.clear()' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.clear()'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.remove(#GENERIC)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.remove(#GENERIC)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.removeAll(kotlin.collections.Collection<#GENERIC>)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.removeAll(kotlin.collections.Collection<#GENERIC>)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.retainOrRemoveAllInternal#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.retainOrRemoveAllInternal#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.indexOf(#GENERIC)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.indexOf(#GENERIC)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.removeAt(kotlin.Int)#GENERIC' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.removeAt(kotlin.Int)#GENERIC'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.removeAtInternal#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.removeAtInternal#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.removeRangeInternal#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.removeRangeInternal#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.addAllInternal#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.addAllInternal#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.copyToArrayImpl$stdlib(kotlin.collections.Collection<#STAR>)kotlin.Array<kotlin.Any?>' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.copyToArrayImpl$stdlib(kotlin.collections.Collection<#STAR>)kotlin.Array<kotlin.Any?>'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.copyToArrayImpl$stdlib(kotlin.collections.Collection<#STAR>;kotlin.Array<#GENERIC>)Generic' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.copyToArrayImpl$stdlib(kotlin.collections.Collection<#STAR>;kotlin.Array<#GENERIC>)Generic'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.copyOfNulls$stdlib@kotlin.Array<#GENERIC>.(kotlin.Int;kotlin.Int)Generic' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.copyOfNulls$stdlib@kotlin.Array<#GENERIC>.(kotlin.Int;kotlin.Int)Generic'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.Itr.previous#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.Itr.previous#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.Itr.set#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.Itr.set#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.Itr.add#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.Itr.add#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.Itr.remove#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.Itr.remove#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.Itr.next#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.Itr.next#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.IllegalStateException.<init>(kotlin.String?)kotlin.IllegalStateException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.IllegalStateException.<init>(kotlin.String?)kotlin.IllegalStateException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.add(kotlin.Int;#GENERIC)' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.add(kotlin.Int;#GENERIC)'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.listIterator(kotlin.Int)kotlin.collections.MutableListIterator<#GENERIC>' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.listIterator(kotlin.Int)kotlin.collections.MutableListIterator<#GENERIC>'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.lastIndexOf(#GENERIC)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.lastIndexOf(#GENERIC)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.addAll(kotlin.Int;kotlin.collections.Collection<#GENERIC>)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.addAll(kotlin.Int;kotlin.collections.Collection<#GENERIC>)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.set(kotlin.Int;#GENERIC)#GENERIC' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.set(kotlin.Int;#GENERIC)#GENERIC'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.subList(kotlin.Int;kotlin.Int)kotlin.collections.MutableList<#GENERIC>' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.subList(kotlin.Int;kotlin.Int)kotlin.collections.MutableList<#GENERIC>'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.get(kotlin.Int)#GENERIC' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.collections.ArrayList.get(kotlin.Int)#GENERIC'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.NumberConverter.freeFormatExponential#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.NumberConverter.freeFormatExponential#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.NumberConverter.Companion.convert(kotlin.Double)kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.NumberConverter.Companion.convert(kotlin.Double)kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.append(kotlin.Double)kotlin.text.StringBuilder' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.StringBuilder.append(kotlin.Double)kotlin.text.StringBuilder'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KTypeImpl.hashCode()ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KTypeImpl.hashCode()ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KTypeImpl.equals(kotlin.Any?)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KTypeImpl.equals(kotlin.Any?)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KFunctionImpl.hashCode()ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KFunctionImpl.hashCode()ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KFunctionImpl.toString()kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KFunctionImpl.toString()kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KFunctionImpl.equals(kotlin.Any?)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KFunctionImpl.equals(kotlin.Any?)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.regex.HangulDecomposedCharSet.<get-decomposedCharUTF16>#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.regex.HangulDecomposedCharSet.<get-decomposedCharUTF16>#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.ArrayIndexOutOfBoundsException.<init>()kotlin.ArrayIndexOutOfBoundsException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.ArrayIndexOutOfBoundsException.<init>()kotlin.ArrayIndexOutOfBoundsException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.ClassCastException.<init>(kotlin.String?)kotlin.ClassCastException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.ClassCastException.<init>(kotlin.String?)kotlin.ClassCastException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.Throwable.<get-stackTraceStrings>#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.Throwable.<get-stackTraceStrings>#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.Throwable.printStackTrace()' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.Throwable.printStackTrace()'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.concurrent.FreezingException.<init>(kotlin.Any;kotlin.Any)kotlin.native.concurrent.FreezingException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.concurrent.FreezingException.<init>(kotlin.Any;kotlin.Any)kotlin.native.concurrent.FreezingException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.ThrowFreezingException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.ThrowFreezingException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.ThrowInvalidMutabilityException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.ThrowInvalidMutabilityException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.concurrent.debugDescription#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.concurrent.debugDescription#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KProperty1Impl.get(#GENERIC)#GENERIC' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KProperty1Impl.get(#GENERIC)#GENERIC'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KProperty1Impl.invoke(#GENERIC)#GENERIC' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KProperty1Impl.invoke(#GENERIC)#GENERIC'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KProperty1Impl.equals(kotlin.Any?)ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KProperty1Impl.equals(kotlin.Any?)ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KProperty1Impl.hashCode()ValueType' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KProperty1Impl.hashCode()ValueType'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.native.internal.KProperty1Impl.toString()kotlin.String' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.native.internal.KProperty1Impl.toString()kotlin.String'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.ThrowArrayIndexOutOfBoundsException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.ThrowArrayIndexOutOfBoundsException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.ThrowClassCastException' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.ThrowClassCastException'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.PrintThrowable' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.PrintThrowable'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.$<get-stackTraceStrings>$FUNCTION_REFERENCE$63.invoke#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.$<get-stackTraceStrings>$FUNCTION_REFERENCE$63.invoke#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.regex.$<get-decomposedCharUTF16>$FUNCTION_REFERENCE$263.invoke#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.regex.$<get-decomposedCharUTF16>$FUNCTION_REFERENCE$263.invoke#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.regex.DecomposedCharSet.<get-decomposedCharUTF16>#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.regex.DecomposedCharSet.<get-decomposedCharUTF16>#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.kfun:kotlin.text.regex.$<get-decomposedCharUTF16>$FUNCTION_REFERENCE$265.invoke#internal' from `app/libapp.a(program.o)' being placed in section `.ARM.extab.text.kfun:kotlin.text.regex.$<get-decomposedCharUTF16>$FUNCTION_REFERENCE$265.invoke#internal'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.note.GNU-stack' from `app/libapp.a(program.o)' being placed in section `.note.GNU-stack'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab' from `~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/thumb/v7-m\libgcc.a(pr-support.o)' being placed in section `.ARM.extab'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.__cxa_begin_catch' from `~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(eh_catch.o)' being placed in section `.ARM.extab.text.__cxa_begin_catch'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.__gxx_personality_v0' from `~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(eh_personality.o)' being placed in section `.ARM.extab.text.__gxx_personality_v0'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text._ZN10__cxxabiv111__terminateEPFvvE' from `~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(eh_terminate.o)' being placed in section `.ARM.extab.text._ZN10__cxxabiv111__terminateEPFvvE'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.__cxa_type_match' from `~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(eh_arm.o)' being placed in section `.ARM.extab.text.__cxa_type_match'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text.__cxa_call_unexpected' from `~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(eh_call.o)' being placed in section `.ARM.extab.text.__cxa_call_unexpected'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text._ZNK10__cxxabiv120__si_class_type_info12__do_dyncastEiNS_17__class_type_info10__sub_kindEPKS1_PKvS4_S6_RNS1_16__dyncast_resultE' from `~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(si_class_type_info.o)' being placed in section `.ARM.extab.text._ZNK10__cxxabiv120__si_class_type_info12__do_dyncastEiNS_17__class_type_info10__sub_kindEPKS1_PKvS4_S6_RNS1_16__dyncast_resultE'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/../../../../arm-none-eabi/bin/ld.exe: warning: orphan section `.ARM.extab.text._ZN9__gnu_cxx27__verbose_terminate_handlerEv' from `~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(vterminate.o)' being placed in section `.ARM.extab.text._ZN9__gnu_cxx27__verbose_terminate_handlerEv'.
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/thumb/v7-m\libgcc.a(unwind-arm.o): In function `unwind_phase2':
unwind-arm.c:(.text+0x1d8): undefined reference to `abort'
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/thumb/v7-m\libgcc.a(unwind-arm.o): In function `__gnu_Unwind_Resume':
unwind-arm.c:(.text+0x35c): undefined reference to `abort'
unwind-arm.c:(.text+0x374): undefined reference to `abort'
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/bin/../lib/gcc/arm-none-eabi/7.2.1/thumb/v7-m\libgcc.a(pr-support.o): In function `_Unwind_GetDataRelBase':
pr-support.c:(.text+0x36a): undefined reference to `abort'
app/libapp.a(program.o): In function `kfun:main(kotlin.Array<kotlin.String>)':
F:\src\kotlin-native\samples\zephyr-hello\src/main.kt:17: undefined reference to `__aeabi_memclr4'
app/libapp.a(program.o): In function `kfun:kotlin.Any.toString()kotlin.String':
F:\src\kotlin-native\runtime\src\main\kotlin\kotlin/Any.kt:41: undefined reference to `__aeabi_memclr4'
app/libapp.a(program.o): In function `kfun:kotlin.native.internal.KClassImpl.<get-qualifiedName>()kotlin.String?':
F:\src\kotlin-native\runtime\src\main\kotlin\kotlin\native\internal/KClassImpl.kt:21: undefined reference to `__aeabi_memclr4'
app/libapp.a(program.o): In function `kfun:kotlin.text.checkRadix$stdlib(kotlin.Int)ValueType':
F:\src\kotlin-native\runtime\src\main\kotlin\kotlin\text/Char.kt:109: undefined reference to `__aeabi_memclr4'
app/libapp.a(program.o): In function `kfun:kotlin.native.concurrent.FreezeAwareLazyImpl.<get-value>()#GENERIC':
F:\src\kotlin-native\runtime\src\main\kotlin\kotlin\native\concurrent/Lazy.kt:23: undefined reference to `__aeabi_memclr4'
app/libapp.a(program.o):F:\src\kotlin-native\runtime\src\main\kotlin\kotlin\native\concurrent/Lock.kt:21: more undefined references to `__aeabi_memclr4' follow
app/libapp.a(program.o): In function `konan::strnlen(char const*, unsigned int)':
:(.text._ZN5konan7strnlenEPKcj+0x0): undefined reference to `strnlen'
app/libapp.a(program.o): In function `konan::consoleErrorUtf8(void const*, unsigned int)':
:(.text._ZN5konan16consoleErrorUtf8EPKvj+0x6): undefined reference to `write'
app/libapp.a(program.o): In function `konan::abort()':
:(.text._ZN5konan5abortEv+0x4): undefined reference to `abort'
app/libapp.a(program.o): In function `InitMemory':
:(.text.InitMemory+0x20): undefined reference to `__aeabi_memclr8'
app/libapp.a(program.o): In function `std::__1::deque<ContainerHeader*, KonanAllocator<ContainerHeader*> >::__add_front_capacity()':
:(.text._ZNSt3__15dequeIP15ContainerHeader14KonanAllocatorIS2_EE20__add_front_capacityEv+0x190): undefined reference to `__aeabi_memmove4'
app/libapp.a(program.o): In function `std::__1::deque<ContainerHeader*, KonanAllocator<ContainerHeader*> >::__add_back_capacity()':
:(.text._ZNSt3__15dequeIP15ContainerHeader14KonanAllocatorIS2_EE19__add_back_capacityEv+0x62): undefined reference to `__aeabi_memmove4'
:(.text._ZNSt3__15dequeIP15ContainerHeader14KonanAllocatorIS2_EE19__add_back_capacityEv+0xea): undefined reference to `__aeabi_memmove4'
:(.text._ZNSt3__15dequeIP15ContainerHeader14KonanAllocatorIS2_EE19__add_back_capacityEv+0x1fc): undefined reference to `__aeabi_memmove4'
app/libapp.a(program.o): In function `std::__1::__split_buffer<ContainerHeader**, KonanAllocator<ContainerHeader**> >::push_back(ContainerHeader**&&)':
:(.text._ZNSt3__114__split_bufferIPP15ContainerHeader14KonanAllocatorIS3_EE9push_backEOS3_+0x4a): undefined reference to `__aeabi_memmove4'
app/libapp.a(program.o)::(.text._ZNSt3__114__split_bufferIPP15ContainerHeader14KonanAllocatorIS3_EE10push_frontEOS3_+0x46): more undefined references to `__aeabi_memmove4' follow
app/libapp.a(program.o): In function `Kotlin_CharArray_copyImpl':
:(.text.Kotlin_CharArray_copyImpl+0x66): undefined reference to `__aeabi_memmove'
app/libapp.a(program.o): In function `Kotlin_io_Console_print':
:(.text.Kotlin_io_Console_print+0xbe): undefined reference to `write'
app/libapp.a(program.o): In function `std::__1::basic_string<char, std::__1::char_traits<char>, KonanAllocator<char> >::push_back(char)':
:(.text._ZNSt3__112basic_stringIcNS_11char_traitsIcEE14KonanAllocatorIcEE9push_backEc+0x90): undefined reference to `__aeabi_memcpy'
app/libapp.a(program.o): In function `Kotlin_io_Console_println':
:(.text.Kotlin_io_Console_println+0x18): undefined reference to `write'
app/libapp.a(program.o): In function `Kotlin_String_fromCharArray':
:(.text.Kotlin_String_fromCharArray+0x5a): undefined reference to `__aeabi_memcpy'
app/libapp.a(program.o): In function `Kotlin_String_plusImpl':
:(.text.Kotlin_String_plusImpl+0x5e): undefined reference to `__aeabi_memcpy'
:(.text.Kotlin_String_plusImpl+0x72): undefined reference to `__aeabi_memcpy'
app/libapp.a(program.o): In function `Kotlin_StringBuilder_insertString':
:(.text.Kotlin_StringBuilder_insertString+0x1e): undefined reference to `__aeabi_memcpy'
app/libapp.a(program.o): In function `Kotlin_String_subSequence':
:(.text.Kotlin_String_subSequence+0x58): undefined reference to `__aeabi_memcpy'
app/libapp.a(program.o): In function `freezeCyclic':
:(.text.freezeCyclic+0x1a8): undefined reference to `ceilf'
app/libapp.a(program.o): In function `std::__1::__hash_table<std::__1::__hash_value_type<ContainerHeader*, std::__1::vector<ContainerHeader*, KonanAllocator<ContainerHeader*> > >, std::__1::__unordered_map_hasher<ContainerHeader*, std::__1::__hash_value_type<ContainerHeader*, std::__1::vector<ContainerHeader*, KonanAllocator<ContainerHeader*> > >, std::__1::hash<ContainerHeader*>, true>, std::__1::__unordered_map_equal<ContainerHeader*, std::__1::__hash_value_type<ContainerHeader*, std::__1::vector<ContainerHeader*, KonanAllocator<ContainerHeader*> > >, std::__1::equal_to<ContainerHeader*>, true>, KonanAllocator<std::__1::__hash_value_type<ContainerHeader*, std::__1::vector<ContainerHeader*, KonanAllocator<ContainerHeader*> > > > >::rehash(unsigned int)':
:(.text._ZNSt3__112__hash_tableINS_17__hash_value_typeIP15ContainerHeaderNS_6vectorIS3_14KonanAllocatorIS3_EEEEENS_22__unordered_map_hasherIS3_S8_NS_4hashIS3_EELb1EEENS_21__unordered_map_equalIS3_S8_NS_8equal_toIS3_EELb1EEES5_IS8_EE6rehashEj+0xa4): undefined reference to `ceilf'
app/libapp.a(program.o): In function `void (anonymous namespace)::traverseContainerReferredObjects<freezeCyclic::$_4>(ContainerHeader*, freezeCyclic::$_4)::{lambda(ObjHeader**)#1}::operator()(ObjHeader**) const':
:(.text._ZZN12_GLOBAL__N_132traverseContainerReferredObjectsIZ12freezeCyclicE3$_4EEvP15ContainerHeaderT_ENKUlPP9ObjHeaderE_clES7_+0x17a): undefined reference to `ceilf'
app/libapp.a(program.o): In function `std::__1::__hash_table<ContainerHeader*, std::__1::hash<ContainerHeader*>, std::__1::equal_to<ContainerHeader*>, KonanAllocator<ContainerHeader*> >::rehash(unsigned int)':
:(.text._ZNSt3__112__hash_tableIP15ContainerHeaderNS_4hashIS2_EENS_8equal_toIS2_EE14KonanAllocatorIS2_EE6rehashEj+0xa4): undefined reference to `ceilf'
app/libapp.a(program.o): In function `Konan_ensureAcyclicAndSet':
:(.text.Konan_ensureAcyclicAndSet+0x206): undefined reference to `ceilf'
app/libapp.a(program.o): In function `simpleShiftLeftHighPrecision':
:(.text.simpleShiftLeftHighPrecision+0x280): undefined reference to `__aeabi_memclr8'
app/libapp.a(program.o): In function `Kotlin_native_NumberConverter_ceil':
:(.text.Kotlin_native_NumberConverter_ceil+0x4): undefined reference to `ceil'
app/libapp.a(program.o): In function `Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl':
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x18): undefined reference to `__aeabi_memclr8'
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x20): undefined reference to `__aeabi_memclr8'
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x28): undefined reference to `__aeabi_memclr8'
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x30): undefined reference to `__aeabi_memclr8'
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x38): undefined reference to `__aeabi_memclr8'
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x16c): undefined reference to `ceil'
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x1ae): undefined reference to `__aeabi_memcpy8'
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x732): undefined reference to `__aeabi_memcpy8'
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x7f4): undefined reference to `__aeabi_memcpy8'
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x8b8): undefined reference to `__aeabi_memcpy8'
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x986): undefined reference to `__aeabi_memcpy8'
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x9c2): undefined reference to `__aeabi_memclr8'
:(.text.Kotlin_native_NumberConverter_bigIntDigitGeneratorInstImpl+0x9ce): undefined reference to `__aeabi_memcpy8'
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(eh_personality.o): In function `read_encoded_value_with_base(unsigned char, unsigned int, unsigned char const*, unsigned int*)':
eh_personality.cc:(.text._ZL28read_encoded_value_with_basehjPKhPj+0xfe): undefined reference to `abort'
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(eh_personality.o): In function `base_of_encoded_value(unsigned char, _Unwind_Context*) [clone .part.3]':
eh_personality.cc:(.text._ZL21base_of_encoded_valuehP15_Unwind_Context.part.3+0x40): undefined reference to `abort'
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(eh_personality.o): In function `__gxx_personality_v0':
eh_personality.cc:(.text.__gxx_personality_v0+0x4a2): undefined reference to `abort'
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(eh_terminate.o): In function `__cxxabiv1::__terminate(void (*)())':
eh_terminate.cc:(.text._ZN10__cxxabiv111__terminateEPFvvE+0x6): undefined reference to `abort'
eh_terminate.cc:(.text._ZN10__cxxabiv111__terminateEPFvvE+0xe): undefined reference to `abort'
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(vterminate.o):vterminate.cc:(.text._ZN9__gnu_cxx27__verbose_terminate_handlerEv+0x8c): more undefined references to `abort' follow
~/.konan/dependencies/gcc-arm-none-eabi-7-2017-q4-major-win32/arm-none-eabi/lib/thumb/libsupc++.a(vterminate.o): In function `__gnu_cxx::__verbose_terminate_handler()':
vterminate.cc:(.text._ZN9__gnu_cxx27__verbose_terminate_handlerEv+0x100): undefined reference to `_impure_ptr'
collect2.exe: error: ld returned 1 exit status
```

Reply:
Maybe just WASM ifdef here https://github.com/JetBrains/kotlin-native/blob/828b2702bf8c72e71f1f1ad674252a836ab67d53/runtime/src/main/cpp/Porting.cpp#L402 shall extend to Zephyr as well. Generally Kotlin/Native runtime shalln't need libc in embedded case.

Reply:
But IMHO it is out of scope for this PR, here better just use `-g` and NEWLIB

Reply:
OK, fine if t will be done in separate PR.